Dockerfile

```diff
@@ -1,19 +1,18 @@
-FROM crystallang/crystal:1.2.2-alpine AS builder
+FROM docker.io/crystallang/crystal:1.6.2-alpine AS builder
```

Comment:
Obsolete due to https://github.com/toddsundsted/ktistec/commit/29f45a08c76ea73f7048772b50c9f71dc0bc5644